### ISSUE TITLE
Integrate a model/view class (Kendall)

### ISSUE THREAD

**Issue by rgarmsen2295** [joncatanio/testtool#11]
1. Implementation of stubbed methods in the model class; there must be at least three such methods for each team member.
2. Design and implementation of one or more companion view classes for the chosen model class, in which there must be at least three action-producing UI elements that call a model method. The GUIs are chosen from the most important aspects of the requirements, as discussed in your team's milestone 4 evaluation.
3. Display of the GUI via the appropriate top-level menu or toolbar command.
   Invocation of the stubbed model methods from the view, to validate that the model/view communication is happening.

To summarize, this deliverable requires that each team member connect at least three model methods to the view, defined in at least one model/view class pair.

(Please update with the model/view class names/file names to confirm files are complete)
